### PR TITLE
Introduce observeAllProperties and unobserveAllProperties

### PR DESCRIPTION
**Pull request by sebastiankb**
see issue https://github.com/w3c/wot-thing-description/issues/800

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1025.html" title="Last updated on Jan 20, 2021, 3:23 PM UTC (0685b02)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1025/48e6fa2...0685b02.html" title="Last updated on Jan 20, 2021, 3:23 PM UTC (0685b02)">Diff</a>

### CODE REVIEW

**Comment by egekorkan** [comment]
Are the corresponding assertions marked at risk or will it be automatic. @mmccool ?

File: index.html [hunk 1]
```diff
@@ -1772,7 +1773,7 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
...
                 be assigned one or more interaction verb(s) each
                 representing a semantic intention of an
-                operation.</td><td>optional</td><td>any type (one of <code>readproperty</code>, <code>writeproperty</code>, <code>observeproperty</code>, <code>unobserveproperty</code>, <code>invokeaction</code>, <code>subscribeevent</code>, <code>unsubscribeevent</code>, <code>readallproperties</code>, <code>writeallproperties</code>, <code>readmultipleproperties</code>, or <code>writemultipleproperties</code>)</td></tr></tbody></table><p>Possible values for the <code>contentCoding</code>
+                operation.</td><td>optional</td><td>any type (one of <code>readproperty</code>, <code>writeproperty</code>, <code>observeproperty</code>, <code>unobserveproperty</code>, <code>invokeaction</code>, <code>subscribeevent</code>, <code>unsubscribeevent</code>, <code>readallproperties</code>, <code>writeallproperties</code>, <code>readmultipleproperties</code>, <code>writemultipleproperties</code>, <code>observeAllProperties</code>, or <code>unobserveAllProperties</code>)</td></tr></tbody></table><p>Possible values for the <code>contentCoding</code>
```
Comment (danielpeintner) [inline index.html:1776]: I think the capitalization in `observeAllProperties` and `unobserveAllProperties` is wrong. There should be **no** capitalization.

Comment (sebastiankb) [reply index.html:1776]: you are right, however, I have no clue why this happen or where I have to change this. @vcharpenay do you have any idea?

File: ontology/td.ttl [hunk 2]
```diff
@@ -211,19 +211,30 @@
...
+    rdfs:comment """Operation type of forms used to read all property values"""@en ;
     rdfs:isDefinedBy : .
 
 :writeAllProperties rdf:type :OperationType ;
     rdfs:label "writeAllProperties" ;
     rdfs:comment """Operation type of forms used to read a property value"""@en ;
     rdfs:isDefinedBy : .
 
+:observeAllbserveProperties rdf:type :OperationType ;
+    rdfs:label "observeAllbserveProperties" ;
```
Comment (danielpeintner) [inline ontology/td.ttl:228]: the text ID's look strange "observeAllbserveProperties"

File: index.html (hunk 1 above)
Comment (vcharpenay) [reply index.html:1776]: lowercase labels are defined in `/td-context/*.jsonld`. That's the part that was missing.

Comment (sebastiankb) [reply index.html]: thanks

**Comment by sebastiankb** [comment]
From today's TD call we will merge this PR.

**Pull request merged**